regex-assembly/932175.ra

```diff
@@ -14,10 +14,12 @@
 
   ##! match white space between command and arguments
   \b\s+
+  \b\s+[+-][a-z]+\s+
+  \b\s+[+-][a-z]+[+]\s+
```

Comment:
```suggestion
  \b\s+[+\-][a-z]+\+\s+
```

Reply:
This is not needed and is automatically created from the regex I had written.
```
SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAMES|ARGS|XML:/* "@rx \ba[\"'\)\[\x5c]*(?:(?:(?:\|\||&&)[\s\x0b]*)?\$[!#\(\*\-0-9\?@_a-\{]*)?\x5c?l[\"'\)\[\x5c]*(?:(?:(?:\|\||&&)[\s\x0b]*)?\$[!#\(\*\-0-9\?@_a-\{]*)?\x5c?i[\"'\)\[\x5c]*(?:(?:(?:\|\||&&)[\s\x0b]*)?\$[!#\(\*\-0-9\?@_a-\{]*)?\x5c?a[\"'\)\[\x5c]*(?:(?:(?:\|\||&&)[\s\x0b]*)?\$[!#\(\*\-0-9\?@_a-\{]*)?\x5c?s\b[\s\x0b]+(?:[\+\-][a-z]+\+?[\s\x0b]+)?[!\"%',-\.0-9@-Z_a-z]+=[^\s\x0b]" \
```

Reply:
No matter it is fixed, let's write correct regexes and do not rely on fixing by some automatic tools.